migrations/mapRolesToId.js

```diff
@@ -0,0 +1,212 @@
+require('dotenv').config({ path: '../.env' });
+const { MongoClient } = require('mongodb');
+//const axios = require('axios');
+const mongoose = require('mongoose');
+console.log(process.env.MONGODB_URL,'process.env.MONGODB_URL')
+
+const dbClient = new MongoClient(process.env.MONGODB_URL);
+const request = require('request');
+
+let ObjectId = mongoose.Types.ObjectId;
+
+async function getSubRolesData(professional_role_id){
+
+
+    var options = {
+      'method': 'GET',
+      'url': `https://saas-qa.tekdinext.com/entity-management/v1/entities/subEntityList/${professional_role_id}?type=professional_subroles`,
```

Comment:
@borkarsaish65 take domain from a variable, so in each environment we can update easily